GitUI/CommandsDialogs/FormCleanupRepository.Designer.cs

```diff
@@ -28,231 +28,299 @@ protected override void Dispose(bool disposing)
         /// </summary>
         private void InitializeComponent()
         {
-            this.Preview = new System.Windows.Forms.Button();
```

Comment:
@merriemcgaw FYI, the new auto-formatting in the designer files is rather unwelcome as it leads to the destructive diffs. It looks like this behaviour is on by default, instead it should be opt-in.

Reply:
Anything I can/should do to address this?

Reply:
You can manually revert the changes, only applying the actual lines changed.
Simplifies the merge with the other clean PR